salt/netapi/rest_tornado/saltnado.py

```diff
@@ -250,10 +250,10 @@ def saltclients(self):
             local_client = salt.client.get_local_client(mopts=self.application.opts)
             # TODO: refreshing clients using cachedict
             SaltClientsMixIn.__saltclients = {
-                'local': local_client.run_job,
+                'local': local_client.run_job_async,
```

Comment:
Ah, it's because it's not a coroutine. But what's the difference between 'local' and 'local_async' then? @cachedout do you have any thoughts?